packages/react-components/lib/robots/robot-panel.tsx

```diff
@@ -37,6 +39,7 @@ export interface RobotPanelProps extends React.HTMLProps<HTMLDivElement> {
 export function RobotPanel({
   paginationOptions,
   verboseRobots,
+  leafletMap,
```

Comment:
You should not couple `RobotPanel` with a map, the panel should just be the panel itself and not be aware of the state of another map component. Such interactions should be handled by a common parent, i.e. `RobotPage`.